packages/dds/tree/src/test/feature-libraries/editable-tree/utilities.spec.ts

```diff
@@ -44,23 +43,10 @@ describe("editable-tree utilities", () => {
 	it("isPrimitive", () => {
 		assert(isPrimitive(int32Schema));
 		assert(isPrimitive(stringSchema));
-		assert(isPrimitive(mapStringSchema));
+		assert(!isPrimitive(mapStringSchema));
 		assert(!isPrimitive(optionalChildSchema));
 	});
 
-	it("isPrimitiveValue", () => {
```

Comment:
these were just moved to the other test file since the function being tested was moved previously.